tests/integration_tests.rs

```diff
@@ -70,6 +70,26 @@ fn run_single_test_success() {
         .success();
 }
 
+#[test]
```

Comment:
### Why aren't there tests for failure cases?

The behavior of `rustlings run` for a `mode="output"` exercise with expected output is to simply run the program without running the "test harness". So I can't actually test the test harness itself :disappointed: I suppose this could be an argument in favor of running the comparison on `rustling run`